### DISCUSSION TITLE
Mac Version

### DISCUSSION THREAD

**Discussion by chinarui-na** [m1dugh/native-sound-mixer#4] [General]
I really like it and look forward to the Mac version.

**Comment by m1dugh** [comment]
Sadly, as I am not a mac User, the mac version might take some time to develop. Therefore, I would appreciate any help to develop the c++ source code for mac users. Feel free to make a PR to help me improving this tool.

**Comment by chinarui-na** [comment]
Hi,
    I’m very happy to use native-sound-mixer. I found if I change volume of session(conference software) to 0. This microphone volume of the conference software selected will change to 0.As a result, other software cannot use this microphone to communicate.Is that so?




| |
17621306765
|
|
***@***.***
|
签名由网易邮箱大师定制


On 07/2/2021 ***@***.***> wrote：

Sadly, as I am not a mac User, the mac version might take some time to develop. Therefore, I would appreciate any help to develop the c++ source code for mac users. Feel free to make a PR to help me improving this tool.

**Reply by m1dugh** [reply]
I guess it's that though windows shall not turn microphone volume to 0 when all sessions are set to 0 but as it is microsoft api we can't do much about it (some tricks might exist). I didn't observe this, happy you pointed it out, funny microsoft api features. Does it happen when multiple sessions are active ? In which case i might have done something wrong with the api
